EIPS/eip-7817.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 7817
```

Comment:
```suggestion
eip: 7531
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Sure. Thanks